apps/vyper/src/app/app.tsx

```diff
@@ -119,7 +119,7 @@ const App = () => {
         <div className="px-3 pt-3 mb-3 w-100">
           <CustomTooltip placement="bottom" tooltipText="Clone Vyper examples. Switch to the File Explorer to see the examples.">
             <Button data-id="add-repository" className="w-100 btn btn-secondary" onClick={() => remixClient.cloneVyperRepo()}>
-              Clone Vyper examples repository
```

Comment:
I think we should generalize this to:
Clone a repo of Vyper examples
- I think that won't be too long.

Reply:
Done